<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeEditor</class>
 <widget class="QMainWindow" name="NodeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="centralLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomIn"/>
   <addaction name="separator"/>
   <addaction name="actionZoomOut"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVideo"/>
   <addaction name="separator"/>
   <addaction name="actionViewport"/>
   <addaction name="separator"/>
   <addaction name="actionGaussianBlure"/>
   <addaction name="separator"/>
   <addaction name="actionGammaCorrection"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionVideo">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionViewport">
   <property name="text">
    <string>Viewport</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>zoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>zoomOut</string>
   </property>
  </action>
  <action name="actionCombine">
   <property name="text">
    <string>Combine</string>
   </property>
  </action>
  <action name="actionWriteFile">
   <property name="text">
    <string>WriteFile</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionGaussianBlure">
   <property name="text">
    <string>GaussianBlure</string>
   </property>
  </action>
  <action name="actionGammaCorrection">
   <property name="text">
    <string>GammaCorrection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
